feature/experimental/bgp/ate_tests/base_bgp_session_parameters/base_bgp_session_parameters_test.go

```diff
@@ -279,17 +293,22 @@ func TestEstablishAndDisconnect(t *testing.T) {
 	t.Log("Check BGP parameters")
 	verifyBgpTelemetry(t, dut)
 
+	// Password authentication is currently disabled as how OC handles BGP password
```

Comment:
@liulk Should we remove the code or keep them in comment state? I don't think we will uncomment these code soon. What's the style guidance here? Thanks.

Reply:
Since we're not supposed to be able to retrieve the password back from the device, let's remove the code. It's just confusing when left as a comment.

Reply:
Removed above mentioned code from script. Thank you.